src/pages/app/parts/swap-widget/TransactionStatusWindow/index.tsx

```diff
@@ -157,17 +195,59 @@ const TransactionStatusWindow = ({
   const setShowLargeDisclaimer = useSetRecoilState(ShowLargeDisclaimer)
   const [userConfirmed, setUserconfirmed] = useState(false)
   const [walletToUse, setWalletToUse] = useState<WalletInterface | null>()
+  const terraWallet = useWallet()
+  const lcdClient = useLCDClient({
+    URL:
+      process.env.REACT_APP_STAGE === "mainnet"
+        ? "https://lcd.terra.dev"
+        : "https://bombay-lcd.terra.dev",
```

Comment:
could replace the url string with `terraConfigMainnet` and `terraConfigTestnet` here

Reply:
done